.orchestra/config/components.yml

```diff
@@ -18,6 +18,7 @@ environment:
   - YARN_RC_FILENAME: $ORCHESTRA_ROOT/share/yarnrc
   - NODE_PATH: $ORCHESTRA_ROOT/node_modules${NODE_PATH:+:${NODE_PATH}}
   - LIBRARY_PATH: /usr/lib/x86_64-linux-gnu
+  - TERMINFO: $ORCHESTRA_ROOT/share/terminfo
```

Comment:
@aleclearmind I was thinking isn't it better if the `ncurses` component provides var this instead?